rllib/algorithms/sac/torch/sac_torch_learner.py

```diff
@@ -42,7 +42,7 @@ class SACTorchLearner(DQNRainbowTorchLearner, SACLearner):
     # TODO (simon): Set different learning rates for optimizers.
     @override(DQNRainbowTorchLearner)
     def configure_optimizers_for_module(
-        self, module_id: ModuleID, config: AlgorithmConfig = None, hps=None
+        self, module_id: ModuleID, config: AlgorithmConfig = None
```

Comment:
AH nice. This thing was still hanging around there and never used.

Reply:
Yeah, got rid of this.